tests/testthat/test-gcalendr.R

```diff
@@ -1,64 +1,57 @@
-context("Read calendar events")
+test_that("Can read events", {
+  skip_if_no_token()
```

Comment:
This is the nicest way to gracefully skip over token-needing tests when no token is available.